crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -230,6 +230,13 @@ impl VisibleEntities {
     }
 }
 
+#[derive(Component, Clone, Debug, Default, Reflect, Deref, DerefMut)]
+#[reflect(Component)]
+pub struct VisibleMeshEntities {
```

Comment:
A few doc comments would be great to be able to differentiate this from `VisibleEntities`